MIGRATION.md

```diff
@@ -10,7 +10,14 @@ The following Sass variables have been removed:
 - `$o-header-image-service-version`
 - `$o-header-image-base-url`
 
-The following Sass mixins have been removed and should be replaced with a single call to `oHeader` and the appropriate `$features` argument.
+The first `oHeader` argument is now an options map `$opts`, rather than `$opts`, to align with other Origami components.
+
+```diff
+-@include oHeader($opts: ('drawer', 'sticky'));
++@include oHeader($opts: ('drawer', 'sticky'));
```

Comment:
I can't see a difference here, is there one?

Reply:
I think there has been a find and replace related dilemma here